sqlbrite/src/main/java/com/squareup/sqlbrite/BriteDatabase.java

```diff
@@ -544,9 +553,18 @@ public void executeAndTrigger(String table, String sql) {
    * @see SQLiteDatabase#execSQL(String, Object[])
    */
   public void executeAndTrigger(String table, String sql, Object... args) {
+    executeAndTrigger(Collections.singleton(table), sql, args);
+  }
+
+  /**
+   * See {@link #executeAndTrigger(String, String, Object...)} for usage. This overload allows for triggering multiple tables.
+   *
+   * @see BriteDatabase#executeAndTrigger(String, String, Object...)
+   */
+  public void executeAndTrigger(Set<String> tables, String sql, Object... args) {
     execute(sql, args);
 
-    sendTableTrigger(Collections.singleton(table));
+    sendTableTrigger(Collections.unmodifiableSet(tables));
```

Comment:
Would you prefer I make defensive copies, or is that not a concern (just pass the set as-is)?